src/spd-notification-worker/src/main/java/ca/bc/gov/open/pssg/rsbc/spd/notification/worker/OutputNotificationConsumer.java

```diff
@@ -78,11 +78,13 @@ public void receiveMessage(OutputNotificationMessage message) {
             logger.debug("attempting to store spd document [{}]", fileInfo.getMetaDataReleaseFileName());
             DpsDocumentRequestBody documentRequestBody = new DpsDocumentRequestBody(sftpProperties.getHost(),
                     fileInfo.getFileId());
+            logger.debug("attempting to store spd document, request: {}", documentRequestBody);
             DpsDocumentResponse documentResponse = documentService.storeDocument(documentRequestBody);
-            logger.info("successfully stored file [{}], id [()]", fileInfo.getMetaDataReleaseFileName());
-
 
             if (documentResponse.getRespCode() == SUCCESS_CODE) {
+
+                logger.info("successfully stored file [{}], id [()]", fileInfo.getMetaDataReleaseFileName(), documentResponse.getGuid());
```

Comment:
Is the file ID not enough here? Is there sensitive data in the filename?

Reply:
nope the file name is not sensitive